octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/editor/EntityModelEditor.java

```diff
@@ -439,16 +454,30 @@ private void createSectionsWithEntityData(FormLayoutSection formSection) {
                 parent = sectionClientRight;
             }
 
-            // Add the pair of labels for field and value
-
+            // Add the pair of labels for field and value                        
             CLabel labelFieldName = new CLabel(parent, SWT.NONE);
             labelFieldName.setText(prettifyLabels(fieldName));
             labelFieldName.setFont(JFaceResources.getFontRegistry().getBold(JFaceResources.DEFAULT_FONT));
-            labelFieldName.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false, 1, 1));
-
+            labelFieldName.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false, 1, 1));  
+        
+            labelFieldName.addPaintListener(new PaintListener() {
+            	@Override
+            	    public void paintControl(PaintEvent arg0) {
+            	    // TODO Auto-generated method stub
+            	    labelFieldName.setBackground(SWTResourceManager.getColor(SWT.COLOR_TRANSPARENT));
+            	    Pattern pattern;
+            	    pattern = new Pattern(arg0.gc.getDevice(), 0,0,0,100, arg0.gc.getDevice().getSystemColor(SWT.COLOR_TRANSPARENT),230, arg0.gc.getDevice().getSystemColor(SWT.COLOR_TRANSPARENT),230);
+            	    arg0.gc.setBackgroundPattern(pattern);
+            	    //arg0.gc.fillGradientRectangle(0, 0, labelFieldName.getBounds().width, labelFieldName.getBounds().height, true);
+            	    }
+
+				
+            });
+            
             TruncatingStyledText labelValue = new TruncatingStyledText(parent, SWT.NONE, truncatedLabelTooltip);
             labelValue.setText(fielValue);
             labelValue.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false, 1, 1));
+            labelValue.setForeground(PlatformUI.getWorkbench().getThemeManager().getCurrentTheme().getColorRegistry().get(JFacePreferences.CONTENT_ASSIST_FOREGROUND_COLOR));

```

Comment:
This foreground and background could be fields of the class, since this long method chain is ugly to see in the code